pkg/manifests/manifests.go

```diff
@@ -39,10 +37,10 @@ const (
 	// generate a cluster-signed certificate and populate the secret.
 	ServingCertSecretAnnotation = "service.alpha.openshift.io/serving-cert-secret-name"
 
-	// OwningClusterIngressLabel should be applied to any objects "owned by" a
-	// clusteringress to aid in selection (especially in cases where an ownerref
+	// OwningIngressControllerLabel should be applied to any objects "owned by" a
+	// ingress controller to aid in selection (especially in cases where an ownerref
 	// can't be established due to namespace boundaries).
-	OwningClusterIngressLabel = "ingress.openshift.io/clusteringress"
+	OwningIngressControllerLabel = "ingresscontroller.operator.openshift.io/owning-ingresscontroller"
```

Comment:
Isn't this inventing a new nonexistent API Group? compare to `ingress.operator.openshift.io/owner` or something

Reply:
What do you mean? `operator.openshift.io` is the API group, and it has an `IngressController` kind (no `Ingress` kind).

Reply:
Looked at 3.x labels and 4.x labels used by other operators and there is no specific convention for owner or finalizer.
Also I think we don't have to be rigid on specific labels/annotations unless we expect some interaction between the operators or some other tools.

Reply:
Agreed, we seem to have flexibility to choose whatever makes sense.  I think `ingresscontroller.operator.openshift.io` makes more sense than either `ingress.openshift.io/clusteringress` or `ingress.operator.openshift.io` because they all look like either `<apigroup>/<kind>` or `<kind>.<apigroup>`, but there is no `Ingress` resource in the `operator.openshift.io` API group, and there is no `ingress.openshift.io` API group or `ClusterIngress` kind anymore whereas there _is_ an `IngressController` kind in the `operator.openshift.io` API group.